packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -592,14 +597,23 @@ export class DbTargetDocument {
 
   constructor(
     /**
-     * The targetId identifying a target.
+     * The targetId identifying a target or 0 for a sentinel row.
      */
     public targetId: TargetId,
     /**
      * The path to the document, as encoded in the key.
      */
-    public path: EncodedResourcePath
-  ) {}
+    public path: EncodedResourcePath,
+    /**
+     *
+     */
+    public sequenceNumber?: ListenSequenceNumber
+  ) {
+    assert(
+      (targetId !== 0) !== (typeof sequenceNumber !== 'undefined'),
```

Comment:
nit: Reduce triple negative to a single negative? :-)  E.g.:

`(targetId === 0) === (sequenceNumber !== undefined)`

Also note that while `typeof sequenceNumber !== 'undefined'` is the most correct way to check for undefined (guaranteed to work even on old browsers where somebody could (accidentally) redefine window.undefined to equal something weird), in our codebase we've chosen to allow the simpler `sequenceNumber !== undefined` check since modern browsers prevent you from redefining undefined.  So feel free to simplify.

Reply:
Done.